### DISCUSSION TITLE
[BUG] 物理机部署1panel并使用frp将1panel暴露到公网后，接口请求失败

### DISCUSSION THREAD

**Discussion by realzolo** [1Panel-dev/1Panel#4751] [Q&A]
### 联系方式

zolo@onezol.com

### 1Panel 版本

v1.10.5-lts

### 问题描述

物理机部署1panel并使用frp将1panel暴露到公网后，接口请求失败。
使用公网地址访问，重启“守护进程”中的程序，接口请求失败。内网地址下正常。
![image](https://github.com/1Panel-dev/1Panel/assets/63219216/9146c894-45e9-4851-be28-9f5abe9bfbfd)

### 重现步骤

将1panel使用frp暴露到公网，重启“守护进程”中的程序

**Comment by ChishFoxcat** [comment]
详细结果呢？？？日志呢？？？
